lib/susedistribution.pm

```diff
@@ -357,15 +362,16 @@ sub activate_console {
         }
     }
 
-    if ($console =~ m/^(.*)-console/) {
-        my ($name, $user) = ($1, $1);
-        if ($name eq 'user') {
-            $user = $testapi::username;
-        }
-        elsif ($name eq 'log') {
-            $user = 'root';
-        }
+    $console =~ m/^(\w+)-(console|virtio-terminal)/;
+    my ($name, $user, $type) = ($1, $1, $2);
```

Comment:
@richiejp I think you introduced warnings that I mentioned in https://progress.opensuse.org/issues/16002 here . Can you please take a look?